tools/debug/debugsymbolviewer.py

```diff
@@ -228,6 +228,13 @@ def print_stack(log_file):
 						continue
 				continue
 
+			if 'Code asserted in IRQ state!' in line:
+				print('[Interrupt Stack]')
+			if 'Code asserted in nested IRQ state!' in line:
+				print('[Interrupt Stack]')
+			if 'User stack:' in line:
+				print('[User stack]')
+
```

Comment:
If SP is corrupt, we will dump all stacks. Are we handling this scenario?
We need to parse and provide the symbols in each stack. Also, it would be better if we clearly mention which stack is currently being printed.

Reply:
All the symbols are parsed in the logs, in case of correct/corrupt SP. 
The type of stack information has been updated.